pkg/apis/rook.io/v1alpha2/types.go

```diff
@@ -47,9 +47,8 @@ type Node struct {
 }
 
 type Device struct {
-	Name     string            `json:"name,omitempty"`
-	FullPath string            // TODO: FullPath to be supported for devices
```

Comment:
So full path is gone now?

Reply:
I think today again was a user asking for `FullPath` to device(s) support, so I would "vote" for keeping/leaving the field as a TODO for now. :slightly_smiling_face: